bazaarvoice/emodb
# EMO-7592: Update docker compose support

## What Are We Doing Here?

In this PR we're updating docker-compose to be able to run the local environment in docker. 

## How to Test and Verify

1. Check out this PR
2. Run `mvn clean install -e -Ddependency-check.skip=true -DskipTests=true  -P docker`
3. Cd to docker/
4. Run `docker-compose up -d`
5. Wait for all containers up with `docker ps`
6. Done

## Risk

### Level 

`Low` 

### Required Testing

`Smoke`, `Regression`, or `Manual`. (All changes except documentation need smoke
testing at a minimum).

### Risk Summary

Add one or a few complete sentences about the possible risks or concerns for
this change.

## Code Review Checklist


build and run.

we inevitably come back to this code it will only take hours to figure out, not
days.

a victim of rampaging consistency, and should be using this course of action. 
We don't have coding standards out yet for this project, so please make sure to address any feedback regarding STYLE so the codebase remains consistent.
